extensions/amp-story/1.0/test/validator-amp-story-page-attachment.html

```diff
@@ -70,6 +70,18 @@ <h1>fill</h1>
         >
       </amp-story-page-attachment>
     </amp-story-page>
+    <amp-story-page id="outlink-cta-button">
+      <amp-story-grid-layer template="vertical">
+        <h1>fill</h1>
+      </amp-story-grid-layer>
+      <amp-story-page-attachment
+        data-title="Page attachment title"
+        layout="nodisplay"
+        data-cta-text="Call To Action"
+        cta-image="https://i.ytimg.com/vi/Wm1fWz-7nLQ/hqdefault_live.jpg"
```

Comment:
It's best to use local assets so we don't depend on images hosted somewhere else, plus any issues with licensing. Since this is a validation test, you can just use "https://website.com/image.png" or any placeholder